src/types/frozenset.rs

```diff
@@ -39,9 +39,16 @@ impl<'py> PyFrozenSetBuilder<'py> {
     }
 
     /// Finish building the set and take ownership of its current value
-    pub fn finalize_bound(self) -> Bound<'py, PyFrozenSet> {
+    pub fn finalize(self) -> Bound<'py, PyFrozenSet> {
         self.py_frozen_set
     }
+
+    /// Deprecated name for [`PyFrozenSetBuilder::finalize`].
+    #[deprecated(note = "use `PyFrozenSetBuilder::finalize` instead")]
```

Comment:
I think we should be consistent and use the "rename" phrasing here as well, same for the others. Also this is missing the version